.circleci/config.yml

```diff
@@ -144,6 +209,7 @@ workflows:
           repo: ${DOCKERHUB_ENDPOINT_REPO}
           requires:
             - build-autoendpoint
+            - test
```

Comment:
Now that the autoendpoint CI changes are in, `test` should be a dependency of `deploy-autopush` as well.